desk/app/bait.hoon

```diff
@@ -88,53 +105,97 @@
     :_  this
     =/  full-line=request-line:server  (parse-request-line:server url.request)
     =/  line
-      ?:  ?=([%lure @ @ *] site.full-line)
+      ?:  ?=([%lure @ *] site.full-line)
         t.site.full-line
       ?:  ?=([@ @ *] site.full-line)
         site.full-line
       !!
     ?+    method.request  (give not-found:gen:server)
-        %'GET'
-      ?:  ?=([%bait %who ~] line)
-        (give (json-response:gen:server s+(scot %p our.bowl)))
-      =/  inviter  (slav %p i.line)
-      =/  token  i.t.line
-      =/  =metadata:reel  (fall (~(get by token-metadata) [inviter token]) *metadata:reel)
-      ?:  ?=([@ @ %metadata ~] line)
-        (give (json-response:gen:server (enjs-metadata metadata)))
-      (give (manx-response:gen:server (landing-page metadata)))
+      %'GET'  (get-request line)
+    ::
         %'POST'
-      =/  inviter  (slav %p i.line)
-      =/  token  i.t.line
       ?~  body.request
+        ~&  "body not found"
         (give not-found:gen:server)
       ?.  =('ship=%7E' (end [3 8] q.u.body.request))
+        ~&  "ship not found in body"
         (give not-found:gen:server)
       =/  joiner  (slav %p (cat 3 '~' (rsh [3 8] q.u.body.request)))
-      :*  :*  %pass  /bite  %agent  [inviter %reel]
-              %poke  %reel-bite  !>([%bite-1 token joiner inviter])
-          ==
-          :*  %pass  /bite  %agent  [our.bowl %reel]
-              %poke  %reel-bite  !>([%bite-1 token joiner inviter])
-          ==
-          (give (manx-response:gen:server (sent-page joiner)))
-      ==
+      =;  [=bite:reel inviter=(unit ship)]
+        ?~  inviter
+          ~&  "inviter not found"
+          (give not-found:gen:server)
+        ^-  (list card)
+        :*  :*  %pass  /bite  %agent  [u.inviter %reel]
+                %poke  %reel-bite  !>(bite)
+            ==
+            :*  %pass  /bite  %agent  [our.bowl %reel]
+                %poke  %reel-bite  !>(bite)
+            ==
```

Comment:
Not entirely sure I remember why we send bites to both. The... group host will be subscribed to our reel, from which it'll get a fact that tells it to send a group invite/let someone in. But then the inviter needs to know, because.. the not-group case?

Reply:
But even there it'll have subscribed through the group-side logic, right?

If we think we can get away with taking it out, I'd take it out. If we don't want to risk it rn, that's fine too, but I could make this with a `::TODO` as something to get a grip on.